packages/@aws-cdk/aws-rds/README.md

```diff
@@ -48,14 +48,14 @@ use the static factory methods on `DatabaseClusterEngine`:
 ```typescript
 new rds.DatabaseCluster(this, 'Database', {
   engine: rds.DatabaseClusterEngine.aurora({
-    version: '5.6.mysql_aurora.1.17.9',
+    version: rds.AuroraEngineVersion.VER_1_17_9, // different version class for each engine type
```

Comment:
Do we need the `VER` prefix? `1_17_9` or `V1_17_9` instead?

Reply:
i like dropping the `VER` prefix - closer to how we do runtimes with Lambda

Reply:
We can't name it `1_17_9`, because variable names in JavaScript (and many other languages ;p) need to start with a letter or underscore.

Reply:
And the problem with just `V` is that there is also a `V` suffix (for example, in `OracleEngineVersion.VER_11_2_0_4_V16`). So that looks a little bit awkward.

Reply:
gotcha, did not know about these limitations